composables/auto-prompter.ts

```diff
@@ -294,25 +301,193 @@ export function useAutoPrompter() {
 
         newBattle.winner = winner
         battles.value.push(newBattle)
+    }
 
-        takeSnapshotOfRatings()
+    // define ammount of battles and prompts in each round
+    function defineBattlesPerRound (numberOfPrompts: number, numberOfBattles: number) {
+        const promptsPerRound: number[] = []
+        const battlesPerRound: number[] = []
+        let sumOfPromptsPerRound = 0
+        let sumOfBattlesPerRound = 0
+
+        // calculate number of prompts that will be alive in each round
+        promptsPerRound[0] = numberOfPrompts
```

Comment:
```suggestion
    // Define amount of battles and prompts in each bracket
    function defineBattlesPerBracket (numberOfBattles: number) {
        const promptsPerBracket: number[] = []
        const battlesPerBracket: number[] = []
        let sumOfPromptsPerBracket = 0
        let sumOfBattlesPerBracket = 0

        // Calculate the number of prompts that will be alive in each bracket
        promptsPerBracket[0] = candidates.value.length
```

We need to change the terminology here to avoid confusion!

What I got so far:

From an initial amount of prompt candidates, we determine the brackets
In each bracket, we'll battle prompts, where the battle's winner is determined by whoever wins the most rounds
Can we move forward with clear definitions?

Also, we don't need to receive the number of prompts as parameters for the function because we already have access to this information in the composable scope.